packages/core/__tests__/utils/project.ts

```diff
@@ -54,9 +85,9 @@ export default class Project {
         skipLibCheck: true,
         allowJs: true,
         checkJs: false,
-        ...compilerOptions,
+        ...(config.compilerOptions ?? {}),
```

Comment:
[nit] I think `null`/`undefined` spread fine on their own?

Reply:
Oh, neat. (In a way, it's neat. 😂)

Reply:
IIRC the justification was that `Object.assign` was already specced in ES2015 to accept falsy values, and there was desire use the same semantics for `{...foo}` instead of allowing subtle differences.

(Which means _instead_ we get the less-subtle difference that `{...null}` is fine but `[...null]` blows up 🙃)

Reply:
de0079f